components/tests/ui/robot_setup.sh

```diff
@@ -57,6 +68,13 @@ do
   bin/omero import -d $delDs $TINY_IMAGE_NAME --debug ERROR
 done
 
+# Import Plate
+bin/omero import $PLATE_NAME --debug ERROR > plate_import.log 2>&1
+plateid=$(sed -n -e 's/^Plate://p' plate_import.log)
+# Use populate_metadata to upload and attach bulk annotation csv
+PYTHONPATH=$PYTHONPATH:lib/python
+python lib/python/omero/util/populate_metadata.py -k $key Plate:$plateid $BULK_ANNOTATION_CSV
```

Comment:
Without something like https://github.com/joshmoore/openmicroscopy/commit/f12a950990d0866bf0ce9393710a25377bcdc567, I'm thinking that this is causing the recent failures of merge-integration (like https://ci.openmicroscopy.org/view/Failing/job/OMERO-5.1-merge-integration/592/). If there are no objections, I'd suggest with close this in favor of gh-3875.